testing/scenario_app/README.md

```diff
@@ -0,0 +1,42 @@
+# Scenario App
+
+This folder contains a dart:ui application and scripts to compile it to AOT
+for exercising embedders.
+
+It intentionally has no dependencies on the Flutter framework or tooling, such
+that it should be buildable as a presubmit or postsubmit to the engine even in
+the face of changes to Dart or dart:ui that require upstream changes in the
+Flutter tooling.
+
+To add a new scenario, create a new subclass of `Scenario` and add it to the
+map in `main.dart`. For an example, see animated_color_square.dart, which draws
+a continuously animating colored square that bounces off the sides of the
+viewport.
+
+## Building for iOS
+
+In this folder, after building the `ios_host` and `ios_profile` engine targets,
+run:
+
+```bash
+./compile_ios_aot.sh ../../../out/host_profile ../../../out/ios_profile/clang_x64/
```

Comment:
uber nit: isn't this more build_mock_framework.sh than compile_ios_aot (which is a subcomponent that happens to mostly build aot as a current implementation detail)?

Reply:
Initially this did just about exactly what `flutter build aot --target-platform=ios` does.  We could rename this if you think it's clearer - this script right now might be a bit overloaded, and ideally we'd have these captured as build rules based on the current build configuration.

Reply:
Let's keep it up to date. If it does more today, let's name it so.